admin/spec/controllers/spree/admin/imports_controller_spec.rb

```
@@ -0,0 +1,121 @@
+
+    before do
+      import.create_mappings
+    end
+
+    it 'renders the show template' do
+      get :show, params: { id: import.id }
+      expect(response).to render_template(:show)
+      expect(assigns(:import)).to eq(import)
+    end
+
+    context 'when mapping is done' do
```

RuboCop (1.81.1)

[convention] Empty example group detected.

(RSpec/EmptyExampleGroup)